ToDoList/src/ToDoList.Api.DependencyInjection/DependencyResolver.cs

```diff
@@ -1,9 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Runtime.CompilerServices;
 using System.Web.Http.Dependencies;
 using Unity;
 using Unity.Exceptions;
 
+[assembly: InternalsVisibleTo("ToDoList.API.DependencyInjection.Tests")]
```

Comment:
Since it is an assembly-wide attribute, please, place it into the `AssemblyInfo.cs`.